### PR TITLE
Machine pool UI

### PR DESCRIPTION
**Pull request by showeimer**

### CODE REVIEW

File: frontend/src/routes/ClusterManagement/ClusterSets/ClusterSets.tsx [hunk 1]
```diff
@@ -263,7 +263,7 @@ export function ClusterSetsTable(props: { clusters?: Cluster[]; managedClusterSe
                             <AcmButton
                                 role="link"
                                 onClick={() => history.push(NavigationPath.createClusterSet)}
-                                disabled={!canCreateClusterSet}
+                                isDisabled={!canCreateClusterSet}
```
Comment (showeimer) [inline frontend/src/routes/ClusterManagement/ClusterSets/ClusterSets.tsx:266]: RBAC bug for create cluster set

File: frontend/src/lib/get-cluster.ts [hunk 2]
```diff
@@ -256,6 +257,13 @@ export function getDistributionInfo(
         }
     }
 
+    if (clusterDeployment) {
+        if (displayVersion === undefined) {
+            const cdVersion = clusterDeployment.metadata.labels?.['hive.openshift.io/version-major-minor-patch']
+            displayVersion = cdVersion ? `OpenShift ${cdVersion}` : undefined
+        }
+    }
+
```
Comment (showeimer) [inline frontend/src/lib/get-cluster.ts:266]: We can display the version from ClusterDeployment if it's a non-managed cluster

File: frontend/src/routes/ClusterManagement/Clusters/ClusterDetails/ClusterMachinePools/ClusterMachinePools.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+            search: 'metadata.name',
+            cell: 'metadata.name',
+        },
+        {
+            header: t('table.machineSetReplicas'),
+            sort: 'status.replicas',
+            search: 'status.replicas',
+            cell: (machinePool: MachinePool) => {
+                return `${machinePool.status!.replicas}/${machinePool.spec!.replicas}`
+            },
```
Comment (showeimer) [inline frontend/src/routes/ClusterManagement/Clusters/ClusterDetails/ClusterMachinePools/ClusterMachinePools.tsx:69]: Need to update, don't know how to represent for auto-scale yet

File: frontend/src/routes/ClusterManagement/Clusters/ClusterDetails/ClusterNodes/ClusterNodes.tsx [hunk 4]
```diff
@@ -90,7 +93,25 @@ export function NodesPoolsTable(props: { nodes: NodeInfo[] }) {
...
+                        role="link"
+                        icon={hasOcpConsole && <ExternalLinkAltIcon />}
+                        iconPosition="right"
+                        onClick={() => {
+                            return window.open(launchUrl, hasOcpConsole ? '_blank' : '_self')
+                        }}
+                    >
+                        {node.name}
+                    </AcmButton>
+                )
```
Comment (showeimer) [inline frontend/src/routes/ClusterManagement/Clusters/ClusterDetails/ClusterNodes/ClusterNodes.tsx:113]: Add node link.  Conditionally launch to OCP console or search

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**